_docs-sources/pipelines/tutorial/index.md

```diff
@@ -1,3 +1,159 @@
 # Single Account Tutorial
 
-<!-- Placeholder for future content for Pipelines v2 -->
+In this tutorial, you’ll walk you through the process of setting up Gruntwork Pipelines in a single AWS account. By the end, you’ll have:
+
+- An automated pipeline for deploying infrastructure changes into an AWS account
+- Two GitHub repositories
+  - `infrastructure-live` — Defines the infrastructure that is deployed to your AWS account
+  - `infrastructure-pipelines` — Contains deployment definitions for your infrastructure
+- IAM role in your AWS account that allows GitHub Actions to assume a role in your AWS account using OIDC
+- An S3 Bucket deployed automatically by Gruntwork Pipelines
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- Permissions to create and administer repositories in GitHub
+- A sandbox or development AWS account
+- Valid [AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for an IAM user with AdministratorAccess to the AWS account mentioned above
+- [Boilerplate](https://github.com/gruntwork-io/boilerplate/releases) installed on your system (requires Gruntwork subscription)
```

Comment:
```suggestion
- [Boilerplate](https://github.com/gruntwork-io/boilerplate#install) installed on your system (requires Gruntwork subscription)
```

Minor suggestion; just because this has clear instructions. Feel free to ignore.